### PR TITLE
Fix for wrong declared maven version

### PR DESCRIPTION
**Pull request by MichaelF25**
Hi again,

thx for releasing v0.3.0 but unfortunately the `${version}` property which is declared in the parent POM is not accessible from it's child if the plugin is used by some external project.
This pull request is fixing this - for upcoming releases you could use the maven versions plugin:
`mvn versions:update-child-modules` after you updated the version of the parent POM. However you still have to manually maintain the version in `es-maven-test/src/test/resources/test-project/pom.xml`.

Additionally I've added a new feature for executing commands after successful start of EventStore.

Br,
Michael

### CODE REVIEW

**Comment by michael-schnell** [comment]
What do you think about a separate "post-start" goal?

**Comment by MichaelF25** [comment]
To be honest I was not sure about the graceful stop of EventStore if the post-start command execution throws some exception, so I did it this way. However, a separater post-start goal sounds even better to me!

About the "failed" return code I was not sure since it depends strongly on the command if a return code other than `0` is an error. On the other hand most maven plugins executing external commands are generating some kind of `MojoExecutionException` if the external command did not return with `0`.

To summarize, I'd update the pull request to introduce a new post-start goal as suggested including exception generation if the external command did not return `0` if that's fine for you.

**Comment by michael-schnell** [comment]
OK, I merged the stuff and moved the code into a new class - Take a look.

**Comment by MichaelF25** [comment]
Tried your changes with my project and works like a charm. Thx! :+1:
